<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="4" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-04-02T15:59:13 IST" name="Default suite" finished-at="2024-04-02T15:59:36 IST" duration-ms="23302">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:parallel.ParallelRun@5c671d7f]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2024-04-02T15:59:13 IST" name="Default test" finished-at="2024-04-02T15:59:36 IST" duration-ms="23302">
      <class name="parallel.ParallelRun">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:parallel.ParallelRun@5c671d7f]" started-at="2024-04-02T15:59:13 IST" name="setUpClass" finished-at="2024-04-02T15:59:14 IST" duration-ms="1221" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@2eada095]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:parallel.ParallelRun@5c671d7f]" started-at="2024-04-02T15:59:14 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-04-02T15:59:30 IST" duration-ms="16083" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA["To check the we are able to login the application with the valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login feature"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:parallel.ParallelRun@5c671d7f]" started-at="2024-04-02T15:59:14 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-04-02T15:59:30 IST" duration-ms="16023" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["To check we are not able to login with invalid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login feature"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at parallel.Login.user_should_be_redirected_to_the_dashboard(Login.java:39)
at ✽.user should be redirected to the dashboard "1Test Automation Practice: Working with Login form"(file:///C:/Users/Hustler/eclipse-workspace/CucumberPOM/src/test/resources/parallel/Login.feature:27)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:parallel.ParallelRun@5c671d7f]" started-at="2024-04-02T15:59:14 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-04-02T15:59:30 IST" duration-ms="16178" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["To check the we are able to login the application with the valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login feature"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at parallel.Login.user_should_be_redirected_to_the_dashboard(Login.java:39)
at ✽.user should be redirected to the dashboard "2Test Automation Practice: Working with Secure Page"(file:///C:/Users/Hustler/eclipse-workspace/CucumberPOM/src/test/resources/parallel/Login.feature:21)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:parallel.ParallelRun@5c671d7f]" started-at="2024-04-02T15:59:14 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-04-02T15:59:30 IST" duration-ms="16286" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA["To check we are not able to login with invalid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login feature"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:parallel.ParallelRun@5c671d7f]" started-at="2024-04-02T15:59:14 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-04-02T15:59:30 IST" duration-ms="16175" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA["To check we are not able to login with invalid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login Functionality"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:parallel.ParallelRun@5c671d7f]" started-at="2024-04-02T15:59:14 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-04-02T15:59:30 IST" duration-ms="15579" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA["To check we are not able to login with invalid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login with 3rd invalid credential"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:parallel.ParallelRun@5c671d7f]" started-at="2024-04-02T15:59:30 IST" name="tearDownClass" finished-at="2024-04-02T15:59:36 IST" duration-ms="5724" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- parallel.ParallelRun -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
